sessions/2019/april/Session_29.md

```diff
@@ -0,0 +1,91 @@
+# SOFTWARE ENGINEERING IN JAVA
+
+## Session 29 (02/04/2019)
+
+- Reviewing sprint project (issue) tasks, solving problems and feedback
+- Reviewing main project tasks and planning next Sprint
+
+## Notes
+
+#### Reviewing sprint project (issue) tasks, solving problems and feedback
+
+Revision were made in order to MR (Merge Request) changes were approved.
+
+The application.properties file was renamed to application.yml in order to use that format. So database configuration was written in the YAML sintax:
+
+```yaml
+spring:
+  jpa.hibernate.ddl-auto: none
+  datasource:
+    url: "jdbc:mysql://127.0.0.1:3306/wallet"
+    username: "${MYSQL_USER}"
```

Comment:
You can set a default value after a `:` if you want:
```suggestion
    username: "${MYSQL_USER:my_default_value}"
```

This can be used so you don't need to provide a default value in development.